specs/segragate_object_specific_flows.adoc

```diff
@@ -0,0 +1,295 @@
+          type: Delete
+	  uuid: 1951e821-7aa9-4a91-8183-e73bc8275b9e
+      	  version: 1
+	StartVolume:
+      	  atoms:
+            - tendrl.gluster_integration.objects.volume.atoms.start
+      	  description: "Start Volume"
+      	  enabled: true
+	  inputs:
+            mandatory:
+              - Volume.volname
+      	  post_run:
+            - tendrl.gluster_integration.objects.volume.atoms.volume_started
+      	  pre_run:
+            - tendrl.gluster_integration.objects.volume.atoms.volume_exists
+      	  run: tendrl.gluster_integration.objects.volume.flows.start_volume.StartVolume
+      	  type: Start
+      	  uuid: 1951e821-7aa9-4a91-8183-e73bc8275b6e
+      	  version: 1
+      attrs:
+        arbiter_count:
+          help: "Arbiter count of volume"
+          type: Integer
+        bricks:
+          help: "List of brick mnt_paths for volume"
+          type: List
+        deleted:
+          help: "Flag is volume is deleted"
+          type: Boolean
+        disperse_count:
+          help: "Disperse count of volume"
+          type: Integer
+        disperse_data_count:
+          help: "Disperse data count of volume"
+          type: Integer
+        force:
+          help: "If force execute the action"
+          type: Boolean
+        redundancy_count:
+          help: "Redundancy count of volume"
+          type: Integer
+        replica_count:
+          help: "Replica count of volume"
+          type: Integer
+        stripe_count:
+          help: "Stripe count of volume"
+          type: Integer
+        transport:
+          help: "Transport type for volume"
+          type: String
+        vol_id:
+          help: "ID of the gluster volume"
+          type: String
+        volname:
+          help: "Name of gluster volume"
+          type: String
+      enabled: true
+      value: clusters/$Tendrl_context.cluster_id/Volumes/$Volume.vol_id/
+tendrl_schema_version: 0.3
+
+Note that the flows specific to volume namely delete volume and start volume
+has not dependency on any other object other than volume.
+
+Alternatives
+------------
+
+None
+
+Data model impact
+-----------------
+
+The flows(only object-specific flows) which are currently placed in the global
+nemespace of a particular entity, will be moved under the namespace of the
+specific object. Example in the "Proposed change" section gives a clear picture.
+
+REST API impact
+---------------
+
+API layer can benefit from this proposed change, as it can now easily figure
+out the set of actions specific to an object and expose them as APIs to the
+end user.
+
+Security impact
+---------------
+
+None
+
+Notifications/Monitoring impact
+-------------------------------
+
+None
+
+Other end user impact
+---------------------
+
+None
+
+Performance Impact
+------------------
+
+None
+
+Other deployer impact
+---------------------
+
+None
+
+Developer impact
+----------------
```

Comment:
Dont use "-----" and '=====" just like that as they have special meaning in .adoc files